core/src/processing/opengl/PSurfaceJOGL.java

```diff
@@ -273,6 +273,15 @@ protected void initGL() {
       } else if (PJOGL.profile == 2) {
         try {
           profile = GLProfile.getGL2ES2();
+
+          // workaround for https://jogamp.org/bugzilla/show_bug.cgi?id=1347
+          if (!profile.isHardwareRasterizer()) {
+            GLProfile hardware = GLProfile.getMaxProgrammable(true);
```

Comment:
Hm, I don't understand - I thought fixedFunc means GL2ES1, whereas we want GL2ES2, which is also what the hardware is capable of?